gcloud/datastore/connection.py

```diff
@@ -493,7 +504,7 @@ def save_entity(self, dataset_id, key_pb, properties,
 
         return False, None
 
-    def delete_entities(self, dataset_id, key_pbs):
+    def delete_entities(self, dataset_id, key_pbs, mutation=None):
```

Comment:
Ditto here. We can hopefully long term just remove `mutation` from `Connection` and have functions like `add_deletes_to_mutation`.